make/push.py

```
@@ -38,29 +44,45 @@ def command(args: Namespace):
+        results.extend(client.images.remove(f"{REPO}:{tag}"))
+
+    results = [x for x in results if x.get("ExitCode") != 0]
+    if results:
+        raise ExceptionGroup(  # noqa: F821
+            "Failed to remove tags",
+            [
+                Exception(
+                    f"{x.get('ExitCode')} {x.get('Errors', 'Unknown')}" for x in results
+                )
+            ],
+        )
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)